# Study 876 — Industry-Relative MAX — the teardown

The head-to-head raw vs industry-relative sort, the Newey-West spread *t*, the quintile monotonicity card, the 20,000-draw sign-flip placebo, the two-era cut, the costed timer, and the synthetic control that proves the adjustment sharpens.

In [1]:
R = {'start': '2010-01-04', 'end': '2026-06-30', 'n_names': 50, 'n_sectors': 8, 'n_months': 198, 'n_spread': 197, 'raw_bps': -104.8, 'raw_t': -2.42, 'raw_t1s': -2.53, 'raw_win': 43, 'raw_sharpe': -0.62, 'raw_p': 0.994, 'adj_bps': -89.7, 'adj_t': -2.51, 'adj_t1s': -2.8, 'adj_win': 46, 'adj_sharpe': -0.69, 'adj_p': 0.998, 'q1': 17.5, 'q2': 13.1, 'q3': 15.45, 'q4': 16.95, 'q5': 28.26, 'placebo_obs': -89.7, 'placebo_mean': -0.19, 'placebo_sd': 32.56, 'placebo_sigma_left': -2.75, 'placebo_left_p': 0.0023, 'placebo_draws': 20000, 'era_early_bps': -47.1, 'era_early_t': -1.26, 'era_early_n': 96, 'era_late_bps': -130.2, 'era_late_t': -2.24, 'era_late_n': 101, 'timer_1_gross': -89.7, 'timer_1_cost': 6.2, 'timer_1_net': -95.9, 'timer_1_t': -3.0, 'timer_1_ann': -11.5, 'timer_5_gross': -89.7, 'timer_5_cost': 14.2, 'timer_5_net': -103.9, 'timer_5_t': -3.25, 'timer_5_ann': -12.5, 'null_mean_t': -0.25, 'null_sd_t': 0.88, 'null_fire': 1, 'planted_raw_t': 11.18, 'planted_adj_t': 20.6, 'fingerprint': '5384b8f7f128'}

## The head-to-head — raw MAX vs industry-relative MAX

Monthly Q1 (low MAX) − Q5 (high MAX) next-month spread, equal-weight.

In [2]:
print(f"raw MAX          : {R['raw_bps']:+.1f} bps/mo  NW t = {R['raw_t']:+.2f}  "
      f"one-sample t = {R['raw_t1s']:+.2f}  win {R['raw_win']}%  Sharpe {R['raw_sharpe']:+.2f}")
print(f"industry-relative: {R['adj_bps']:+.1f} bps/mo  NW t = {R['adj_t']:+.2f}  "
      f"one-sample t = {R['adj_t1s']:+.2f}  win {R['adj_win']}%  Sharpe {R['adj_sharpe']:+.2f}")
print('  -> both NEGATIVE (wrong sign vs the claim); adjustment sharpens |t| slightly')

raw MAX          : -104.8 bps/mo  NW t = -2.42  one-sample t = -2.53  win 43%  Sharpe -0.62
industry-relative: -89.7 bps/mo  NW t = -2.51  one-sample t = -2.80  win 46%  Sharpe -0.69
  -> both NEGATIVE (wrong sign vs the claim); adjustment sharpens |t| slightly


## Quintile monotonicity — industry-relative MAX (annualised mean return)

Q5 (the lottery, high-MAX names) posts the *highest* return — the effect runs backwards on mega-caps.

In [3]:
for q in ('q1','q2','q3','q4','q5'):
    print(f"  Q{q[-1]}: {R[q]:+.2f}%/yr")
print(f"  Q5 - Q1 = {R['q5']-R['q1']:+.2f}%/yr (high-MAX out-earns low-MAX)")

  Q1: +17.50%/yr
  Q2: +13.10%/yr
  Q3: +15.45%/yr
  Q4: +16.95%/yr
  Q5: +28.26%/yr
  Q5 - Q1 = +10.76%/yr (high-MAX out-earns low-MAX)


## Placebo — sign-flip null on the industry-relative spread (20,000 draws)

In [4]:
print(f"observed {R['placebo_obs']:+.1f} bps vs placebo mean {R['placebo_mean']:+.2f} "
      f"(sd {R['placebo_sd']:.1f}) -> ~{R['placebo_sigma_left']:+.2f} sigma, left-tail p = {R['placebo_left_p']:.4f}")

observed -89.7 bps vs placebo mean -0.19 (sd 32.6) -> ~-2.75 sigma, left-tail p = 0.0023


## Robustness — two eras (split 2018-01-01)

In [5]:
print(f"2010-2017 (n={R['era_early_n']}): {R['era_early_bps']:+.1f} bps  NW t = {R['era_early_t']:+.2f}  (not significant)")
print(f"2018-2026 (n={R['era_late_n']}): {R['era_late_bps']:+.1f} bps  NW t = {R['era_late_t']:+.2f}")
print('  -> the wrong-sign relation is a LATE-era phenomenon; it fails the sub-era robustness bar')

2010-2017 (n=96): -47.1 bps  NW t = -1.26  (not significant)
2018-2026 (n=101): -130.2 bps  NW t = -2.24
  -> the wrong-sign relation is a LATE-era phenomenon; it fails the sub-era robustness bar


## The timer — can you get paid for it?

Long low-MAX / short high-MAX (industry-relative), 2 sides × one-way × NAV per monthly rebalance; short pays 50 bps/yr borrow.

In [6]:
for tag,g,c,n,t,a in [('1 bp',R['timer_1_gross'],R['timer_1_cost'],R['timer_1_net'],R['timer_1_t'],R['timer_1_ann']),
                      ('5 bps',R['timer_5_gross'],R['timer_5_cost'],R['timer_5_net'],R['timer_5_t'],R['timer_5_ann'])]:
    print(f"{tag:>5} one-way: gross {g:+.1f} -> net {n:+.1f} bps/mo (cost {c:.1f}/mo, t={t:+.2f}, ~{a:+.1f}%/yr)")
print('  the specified book loses money gross AND net at every cost -> Mirage')

 1 bp one-way: gross -89.7 -> net -95.9 bps/mo (cost 6.2/mo, t=-3.00, ~-11.5%/yr)
5 bps one-way: gross -89.7 -> net -103.9 bps/mo (cost 14.2/mo, t=-3.25, ~-12.5%/yr)
  the specified book loses money gross AND net at every cost -> Mirage


## Synthetic positive control — the machinery is unbiased AND the thesis holds

Live: neither sort fires on the null; on the planted world the industry adjustment recovers the effect MORE sharply than raw MAX.

In [7]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
import numpy as np
from max_industry import data, strategy as st
null_t = np.array([st.synthetic_detect(data.synthetic_panel(edge=0.0, seed=876+s, n_months=180), adjusted=True)['t_nw'] for s in range(8)])
print(f"null (edge=0), 8 seeds [ind-rel]: NW t mean {null_t.mean():+.2f} (sd {null_t.std(ddof=1):.2f}), |t|>=2 in {(abs(null_t)>=2).sum()}/8")
planted = data.synthetic_panel(edge=0.012, seed=876, n_months=240)
print(f"planted (edge=0.012): raw-MAX NW t = {st.synthetic_detect(planted, adjusted=False)['t_nw']:+.2f}  vs  ind-rel NW t = {st.synthetic_detect(planted, adjusted=True)['t_nw']:+.2f}")

null (edge=0), 8 seeds [ind-rel]: NW t mean -0.67 (sd 0.80), |t|>=2 in 1/8


planted (edge=0.012): raw-MAX NW t = +11.18  vs  ind-rel NW t = +20.60


## Verdict

- **Signal — None.** The industry-relative MAX does **not** sharpen a valid MAX effect on 50 liquid US mega-caps — because the effect is *inverted* here. The Q1−Q5 spread is **-89.7 bps/mo** (NW *t* = **-2.51**), significant but *opposite in sign* to the claim (the sign-flip placebo puts it ~2.8σ into the left tail), and it holds only in the late era (*t* = -1.26 early / -2.24 late). A significant wrong-sign result **fails the claim**. The 20-seed synthetic control recovers a *planted* relation cleanly and confirms the adjustment sharpens it (raw *t* = +11.18 → ind-rel *t* = +20.60), so the sign-reversal is real, not machinery. Survivorship biases the magnitude.
- **Tradability — Mirage.** The specified long-low / short-high book loses money gross (**-89.7 bps/mo**) and net (**-95.9** at 1 bp, **-103.9** at 5 bps). A Mirage in the claimed direction.